sympy/codegen/cfunctions.py

```diff
@@ -9,7 +9,7 @@
 import math
 from sympy.core.singleton import S
 from sympy.core.numbers import Rational
-from sympy.core.function import ArgumentIndexError, Function, Lambda
+from sympy.core.function import ArgumentIndexError, Function, Lambda, expand_log
```

Comment:
A minor remark: this import is not used anywhere.

Reply:
@gxyd but removing this import will produce `NameError`

Reply:
Why so?

I don't think `expand_log` is used anywhere.

Reply:
@gxyd sorry for this silly mistake, I forgot to remove it when I added in line 100